### PR TITLE
🚀Preview Release

### LINKED ISSUE #441: 🚀Preview Release
### Pre-Release ToDo List

- [X] All issues in the assigned milestone are closed and all issue tasks are complete.
- [X] Add `🚀Preview Release` label to this issue.
- [X] Issue is assigned to a project.
- [X] Issue is assigned to a milestone.
- [x] Release PR linked to this issue.
- [x] All unit tests have been executed locally and have passed. _(Check out the appropriate release branch before running tests)_.
- [x] Version in project file updated _(All changes made directly on the release branch (preview/v#.#.#-preview.#))_.
- [x] Release notes created and added _(All changes made directly on the release branch (preview/v#.#.#-preview.#))_.
- [x] Manual QA Testing completed _(if applicable)_.
- [ ] Pull request has been approved and merged into a release branch before performing the release. _(Releases are performed on preview branches)_.
- [ ] Release to **_preview_** complete. _(The release is performed by running the `🚀Preview Release` workflow)_.

### Post-Release ToDo List

- [ ] The GitHub release has been created and is correct.
- [ ] The NuGet package has been successfully deployed to [nuget.org](https://www.nuget.org/) _(if applicable)_.
- [ ] Announcement of release on [Twitter](https://twitter.com/KDCoder) verified _(if applicable)_. _(Announcement should be performed automatically with the release)_.
- [ ] Announcement has been pushed to the [Discord](https://discord.gg/qewu6fNgv7) channel _(if applicable)_.

### Additional Information:

**_<details closed><summary>Unit Tests</summary>_**

Reasons for local unit test execution:
- Unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
- Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
</details>

---

**_<details closed><summary>Version Updating</summary>_**

The version can be updated by setting the values of the `<Version/>` and `<FileVersion/>` XML tags in the project file.
The `<Version/>` and `<FileVersion/>` values can hold production and preview releases.
The `<AssemblyVersion/>` XML tag can only hold production values.  Preview values are not allowed.

``` xml
<!--Preview Release Example-->
<Version>1.2.3-preview.4</Version>
<FileVersion>1.2.3-preview.4</FileVersion>
<AssemblyVersion>1.2.3</AssemblyVersion>
```
</details>

---

**_<details closed><summary>Release Notes</summary>_**

By default, the release notes go into a **_ReleaseNotes_** folder inside of the **_Documentation_** folder with the **_Documentation_** folder at the root of the repository.
Preview release notes path: **_./Documentation/ReleaseNotes/PreviewReleases_**.

Release note file names must follow a particular syntax and are in markdown format so they can be added to the release.

#### Release Notes File Name Syntax:
- **Preview Release Notes:**
  - **Syntax:** Release-Notes-v#.#.#-preview.#.md
  - **Example:** Release-Notes-v1.2.3-preview.4.md

Changes such as release notes and version updates should be committed to the destination branch (preview/v#.#.#-preview.#) in the pull request attached to this issue.
</details>

---

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Release Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

- [x] The **_base(target)_** branch for this pull request is a **_release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: release/v<major>.<minor>.<patch>
      Example: release/v1.2.3
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_release_** branch with the correct naming syntax in the following example.
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: release/v<major>.<minor>.<patch>
    Example: release/v1.2.3
    ```
  </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title is set to _**🚀Preview Release**_.

- [x] A `🚀Preview Release` label has been added to this pull request. 

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] A QA issue has been created and manual QA testing has been performed for the changes being released _(if applicable)_.

### CODE REVIEW

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    > The layer parameter represents the layer that the texture will be rendered on.  Smaller layers will be rendered
+    > behind layers with a higher layer value.  If 2 textures have the same layer, the order of rendering will be based
+    > on the order of the `Render` method calls.  Negative layer values are allowed.
+    - `Render(RectShape rectangle, int layer = 0)`
+
+---
+
+<h2 style="font-weight:bold" align="center">Bug Fixes 🐛</h2>
+
+1. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Fixed bug where the default size value of one of the render text overloads was incorrect.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md:51]: Grammar
```suggestion
1. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Fixed a bug where the default size value of one of the render text overloads was incorrect.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   - `SetBottomLeft`
+   - `SetBottomRight`
+   - `SetTopRight`
+
+---
+
+<h2 style="font-weight:bold" align="center">Internal Changes ⚙️</h2>
+<h5 align="center">(Changes that do not affect users.  Not breaking changes, new features, or bug fixes.)</h5>
+
+1. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Changed how various batch management service batch size updates were managed.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md:70]: sentence structure
```suggestion
1. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Changed how various batch management service updates were managed.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+3. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Refactored and cleaned up various areas of the code base.
+4. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Refactored, improved, and increased code coverage of unit tests across the code base.
+5. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Created 3 new scenes in the **_Velaptor Testing_** application to test out layered rendering for textures, text, and rectangle primitives.
+6. [#434](https://github.com/KinsonDigital/Velaptor/issues/434) - Made the following improvements in the rectangle scene in the **Velaptor Testing_** application.
+   - Removed the buttons on the left side that move the rectangle around the screen.
+   - Added instruction text to the top window.
+   - Added the ability to move the rectangle around the window using the keyboard **_Arrow Keys_**.
+   - Added the ability to change the size of the rectangle using the keyboard **_Shift_** + **_Arrow Keys_**.
+7. [#438](https://github.com/KinsonDigital/Velaptor/issues/438) - Made simple grammar change to the project README file.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md:94]: Reword
```suggestion
7. [#438](https://github.com/KinsonDigital/Velaptor/issues/438) - Made grammar changes to the project README file.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+8. [#250](https://github.com/KinsonDigital/Velaptor/issues/250) - Updated rectangle scene in the **_Velaptor Testing_** application to automatically adjust horizontal spacing for the buttons at the bottom of the window.
+9. [#292](https://github.com/KinsonDigital/Velaptor/issues/292) - Updated demo section of the project README with a link to a youtube [video](https://www.youtube.com/watch?v=nNeVKvkbXc4) of the features.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.13.md:96]: Spelling
```suggestion
9. [#292](https://github.com/KinsonDigital/Velaptor/issues/292) - Updated demo section of the project README with a link to a YouTube [video](https://www.youtube.com/watch?v=nNeVKvkbXc4) of the features.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Pull request merged**